easybuild/easyconfigs/o/OpenMPI/OpenMPI-5.0.3-GCC-13.3.0.eb

```diff
@@ -32,12 +32,9 @@ dependencies = [
 
 # CUDA related patches and custom configure option can be removed if CUDA support isn't wanted.
 preconfigopts = 'gcc -Iopal/mca/cuda/include -shared opal/mca/cuda/lib/cuda.c -o opal/mca/cuda/lib/libcuda.so && '
-configopts = '--with-cuda=%(start_dir)s/opal/mca/cuda '
+configopts = '--with-cuda=%(start_dir)s/opal/mca/cuda --with-show-load-errors=no '
 
 # disable MPI1 compatibility for now, see what breaks...
 # configopts += '--enable-mpi1-compatibility '
 
```

Comment:
We've had this in as at least as far back as https://github.com/easybuilders/easybuild-easyconfigs/pull/10483 and I am not aware we've found anything it breaks.

Time to remove?

Reply:
Yes, that makes sense. Done in https://github.com/boegel/easybuild-easyconfigs/pull/97/commits/2fb792e55dc308985e9d0c836fa660d64d304ead